tests/unit/test_next_command.py

```
@@ -2,212 +1,549 @@
+        "write_lock": mock_wlock,
+        "apply_context7": mock_context7,
+        "generate_text": mock_gen_text,
+        "utils_ROOT": mock_root_util,
+    }
+
+    patch.stopall() # Stop all patches started with start()
+
+
+# Test Scenarios for run_next()
+
+@pytest.mark.parametrize("mock_config", [pytest.lazy_fixture("mock_ai_provider_openai_direct")], indirect=True)
```

Pylint (3.3.7)

[error] Module 'pytest' has no 'lazy_fixture' member

(E1101)